library/include/rocwmma/internal/wmma.hpp

```diff
@@ -310,7 +308,6 @@ namespace rocwmma
 
                 for(int j = 0; j < upperSrcAIt.range(); j++)
                 {
-                    //TODO : Verify permute
                     Permute<PermuteOps::BlockBCast16<0>>::exec(*lowerSrcAIt, detail::laneId());
```

Comment:
Can operate permute directly on regsAUpper and regsALower:

Permute<PermuteOps::BlockBCast16<0>>::exec(regsAUpper);
Permute<PermuteOps::BlockBCast16<1>>::exec(regsALower);